Calibration/IsolatedParticles/plugins/StudyHLT.cc

```diff
@@ -1,35 +1,137 @@
-#include "StudyHLT.h"
+// system include files
+#include <memory>
+#include <string>
 
-//Triggers
-#include "DataFormats/L1GlobalTrigger/interface/L1GlobalTriggerReadoutRecord.h"
-#include "DataFormats/HLTReco/interface/TriggerObject.h"
-#include "FWCore/Common/interface/TriggerNames.h"
-#include "FWCore/MessageLogger/interface/MessageLogger.h"
-
-#include "DataFormats/DetId/interface/DetId.h"
-#include "DataFormats/EcalDetId/interface/EBDetId.h"
-#include "DataFormats/EcalDetId/interface/EEDetId.h"
-#include "DataFormats/HcalDetId/interface/HcalDetId.h"
+// Root objects
+#include "TH1.h"
+#include "TH2.h"
+#include "TTree.h"
 
+// user include files
 #include "Calibration/IsolatedParticles/interface/FindCaloHit.h"
 #include "Calibration/IsolatedParticles/interface/CaloPropagateTrack.h"
 #include "Calibration/IsolatedParticles/interface/ChargeIsolation.h"
 #include "Calibration/IsolatedParticles/interface/eECALMatrix.h"
 #include "Calibration/IsolatedParticles/interface/eHCALMatrix.h"
+#include "Calibration/IsolatedParticles/interface/TrackSelection.h"
```

Comment:
@mmusich They are indeed needed - they have been there divided into .h and .cc files. Now they are combined into one (I have removed the .h file).